### DISCUSSION TITLE
How to paste previous item?

### DISCUSSION THREAD

**Discussion by kayanco** [hluk/CopyQ#2357] [Q&A]
When I do Ctrl+V, it pastes item in row1.
Is there a way to quickly paste item in row 2? e.g. with Ctrl+Alt+P.

Currently I open copyq window, double-click row2 item, then Ctrl+V to paste it.

**Comment by hluk** [comment]
The following command pastes the second row on Ctrl+Alt+P ([here is how to add the command to CopyQ](http://copyq.readthedocs.io/en/latest/faq.html#how-to-load-shared-commands-and-share-them)):

```ini
[Command]
Command="
    copyq:
    select(1)
    paste()"
GlobalShortcut=ctrl+alt+p
Icon=\xf0ea
IsGlobalShortcut=true
Name=Paste Previous
```
